site/docs/relations/physical_relations.md

```diff
@@ -67,10 +67,35 @@ The merge equijoin does a join by taking advantage of two sets that are sorted o
 | Left Input          | A relational input.                                                                                                                                                                                                     | Required                    |
 | Right Input         | A relational input.                                                                                                                                                                                                     | Required                    |
 | Left Keys           | References to the fields to join on in the left input.                                                                                                                                                                  | Required                    |
-| Right Keys          | References to the fields to join on in the right input.                                                                                                                                                            | Reauired                    |    
+| Right Keys          | References to the fields to join on in the right input.                                                                                                                                                                 | Required                    |
 | Post Join Predicate | An additional expression that can be used to reduce the output of the join operation post the equality condition. Minimizes the overhead of secondary join conditions that cannot be evaluated using the equijoin keys. | Optional, defaults true.    |
 | Join Type           | One of the join types defined in the Join operator.                                                                                                                                                                     | Required                    |
 
+
+
+## Single Join Operator
+
+The single match join works by enforcing that each row is involved in only a single match.  If this violated, an error
+is raised.  Typical use would be to have one side's duplicates eliminated and then be pushed into a column scan on the
+other side. This operator is useful when converting subqueries into joins.
```

Comment:
I'm probably dense but I don't understand how this is helpful.  This doesn't seem like it would be useful in eliminating duplicates (an error would be raised).  Are you saying this is somehow used on the deduplicated values?  How so?  What does "pushed into a column scan on the other side" mean?